VariantShredding.md

```diff
@@ -23,7 +23,7 @@
 > **This specification is still under active development, and has not been formally adopted.**
 
 The Variant type is designed to store and process semi-structured data efficiently, even with heterogeneous values.
-Query engines encode each Variant value in a self-describing format, and store it as a group containing `value` and `metadata` binary fields in Parquet.
+Query engines encode each Variant value in a self-describing format, and store it as a group containing required `value` and `metadata` binary fields in Parquet.
 Since data is often partially homogenous, it can be beneficial to extract certain fields into separate Parquet columns to further improve performance.
```

Comment:
```suggestion
Since Variant values within a column are often partially homogenous, common fields can be extracted into typed Parquet columns in a process defined as **shredding**. An engine utilizing these columns can get all the benefits of structured data while only incurring a performance penalty when accessing the non-shredded elements.
```

Reply:
Suggestion for summarizing the following sections on motivation.